src/test/java/jewellery/inventory/unit/service/ProductServiceTest.java

```diff
@@ -258,7 +259,7 @@ void testDeleteProductShouldThrowExceptionWhenProductIsPartOfProduct() {
 
   @Test
   void testDeleteProductShouldThrowExceptionWhenProductIsSold() {
-    product.setSold(true);
+    product.setPartOfSale(new Sale()); //TODO
```

Comment:
Is there something TODO here? If there is, 
![](https://media.giphy.com/media/3o84sw9CmwYpAnRRni/giphy.gif?cid=ecf05e4729uzkkl99d85y09rati4lp0qh2vj5qiw7veyi63d&ep=v1_gifs_search&rid=giphy.gif&ct=g)
If not, remove the comment.

Reply:
^